helm/policies-common/tests/abs/ensure.py

```diff
@@ -0,0 +1,112 @@
+import yaml
+from functools import partial
+import time
+import random
+import string
+from textwrap import dedent
+
+import pytest
+from pytest_kube import forward_requests, wait_for_rollout, app_template
+
+import logging
+LOGGER = logging.getLogger(__name__)
+
+
+def release(kubectl, release_version, cluster_operator_version, capi_core_version):
+    r = dedent(f"""
+        apiVersion: release.giantswarm.io/v1alpha1
```

Comment:
Thought: not for this PR but to consider for later -- what about pulling these CRs out into their own files and reading them in? May be clearer and easier for others to work with

Reply:
I'll try this in a follow up. Would make managing whitespace nicer.